pkg/arvo/sur/publish.hoon

```diff
@@ -2,15 +2,12 @@
 |%
 ::
 +$  group-info
-  $%  [%old write-pax=path read-pax=path]
-      $:  %new
-          write-grp=(set ship)  write-pax=path
-          read-grp=(set ship)   read-pax=path
-          sec=rw-security
-      ==
+  $:  group-path=path
+      invitees=(set ship)
+      use-preexisting=?
```

Comment:
Going to mention that these flags aren't necessary for posterity, but more style than anything.